hooks/index.ts

```diff
@@ -16,6 +16,7 @@ export { useInterval } from "./helpers/useInterval";
 export { useIsomorphicLayoutEffect } from "./helpers/useIsomorphicLayoutEffect";
 export { useMounted } from "./helpers/useMounted";
 export { usePanelWidth } from "./helpers/usePanelWidth";
+export { useSign } from "./helpers/useSign";
```

Comment:
my editor organized these in alphabetical order, I guess its nice so left it like that